enhancements/management-workload-partitioning.md

```diff
@@ -260,6 +260,13 @@ unschedulable workloads, but we expect the resulting error message to
 explain the problem well enough for admins to recover. Future work may
 make this more flexible.
 
+In a failure situation, we want to try to keep the cluster
+operational. Therefore, when pods ask for workload partitioning but
+the feature is disabled the admission hook will strip the
+workload-related annotations and add an annotation
+`workload.openshift.io/warning` with a message warning the user that
+their partitioning instructions were ignored.
+
```

Comment:
The following scenarios are not mutated and would generate the warning annotation: 
1) Guaranteed QoS class pods 
2) Mutation that would change the QoS Policy

Reply:
It also drops an error when not all nodes under the cluster have management resources.

Reply:
@browsell @cynepco3hahue I have updated the wording about warning handling based on your comments. Please give it another look.